index.js

```diff
@@ -4,7 +4,7 @@ class TodoItem {
     this.completed = completed;
     this.id = id;
   }
-}
+};
 
 class TodoArray {
   constructor() {

```

Comment:
- You respected GitHub flow
- Your codes are dry,  repetition is not available
- The repo has been documented professionally
- Changes can be made but it seems, I can't highlight them to you here in your PR,

Reply:
Thank you for the comment 🙏

Reply:
Can i go ahead to merge this PR